packages/b/buildah/package.yml

```diff
@@ -18,7 +18,6 @@ description: |
     - delete a working container or an image
     - rename a local container
 builddeps  :
-    - pkgconfig(devmapper)
```

Comment:
Is this dependency no longer used?

Reply:
yes, they removed support

Reply:
https://github.com/containers/buildah/pull/4832

Reply:
Nice, thanks